mod_ci/controllers.py

```diff
@@ -564,6 +564,17 @@ def start_ci():
                 except KeyError:
                     g.log.critical("Didn't find a SHA value for a newly opened PR!")
                     g.log.debug(payload)
+                # Check if user blacklisted
+                if BlockedUsers.query.filter(BlockedUsers.userID == payload[
+                    'pull_request']['user']['id']).first() is not None:
+                    log.critical("User Blacklisted")
```

Comment:
I guess it'd be helpful to also log user ID here.

Reply:
Maybe, but as it won't occur a lot I don't see the need immediately.

Reply:
And I'd rather not pollute the db with canceled tests of blocked users.

Reply:
Github notification is enough, if someone ask, we could see that posting link.